e2e/backend.e2e.spec.ts

```diff
@@ -145,9 +154,14 @@ test.describe.serial('KubernetesBackend', () => {
         'options.flannel': false,
       };
 
-      if (os.platform() === 'win32') {
-        expectedDefinition.hostResolver = false;
-      } else {
+      const platformExpectedDefinitions: Partial<Record<NodeJS.Platform, Partial<Record<keyof RecursiveTypes<KubeSettings>, boolean>>>> = {
```

Comment:
Again, I'd like a type definition with example (if in a test) for types that are this long.